src/Orchard/Environment/Extensions/Folders/IExtensionFolders.cs

```diff
@@ -4,5 +4,6 @@
 namespace Orchard.Environment.Extensions.Folders {
     public interface IExtensionFolders {
         IEnumerable<ExtensionDescriptor> AvailableExtensions();
+        IEnumerable<string> Paths { get; }

```

Comment:
Modules and themes settings may change according to tenant.
There are several lists that are used' quit inconsistently around the code which include (If I am not missing any):
List of view lookup folders, list of dependency folders (excluding core), list of folders that should not be translated in tenant href.

It is possible that the best solution would be to define all such lists in some tenant-related settings object.

Where would you suggest?